src/Entity/VotePlace.php

```diff
@@ -0,0 +1,157 @@
+<?php
+
+namespace AppBundle\Entity;
+
+use Doctrine\ORM\Mapping as ORM;
+use Symfony\Bridge\Doctrine\Validator\Constraints\UniqueEntity;
+use Symfony\Component\Validator\Constraints as Assert;
+use Algolia\AlgoliaSearchBundle\Mapping\Annotation as Algolia;
+use AppBundle\Validator\UnitedNationsCountry as AssertUnitedNationsCountry;
+
+/**
+ * @ORM\Table(name="vote_place")
+ * @ORM\Entity
+ *
+ * @UniqueEntity(fields={"code"})
+ *
+ * @Algolia\Index(autoIndex=false)
+ */
+class VotePlace
+{
+    use EntityTimestampableTrait;
+
+    /**
+     * @var int
+     *
+     * @ORM\Column(name="id", type="integer")
+     * @ORM\Id
+     * @ORM\GeneratedValue
+     */
+    private $id;
+
+    /**
+     * @var string
+     *
+     * @ORM\Column
+     *
+     * @Assert\NotBlank
+     * @Assert\Length(max=255)
+     */
+    private $name;
+
+    /**
+     * @var string
+     *
+     * @ORM\Column(length=10, unique=true)
+     *
+     * @Assert\NotBlank
+     * @Assert\Length(max=10)
+     */
+    private $code;
+
+    /**
+     * @var string
+     *
+     * @ORM\Column(length=150)
+     *
+     * @Assert\NotBlank(message="common.address.required")
+     * @Assert\Length(max=150, maxMessage="common.address.max_length")
+     */
+    private $address;
+
+    /**
+     * @var string
+     *
+     * @ORM\Column(type="text", nullable=true)
```

Comment:
`text`? why?

Reply:
because we could have more than one postal code here, will be used with `FIND_IN_SET`